data.md

```diff
@@ -0,0 +1,4 @@
+## Data Retention Policy
+Our BigBlueButton instance keeps your uploaded presentations for 5 days after you have ended your meeting. BigBlueButton log history are kept for 28 days. Currently our BigBlueButton server is not backed up.
```

Comment:
Maybe we should add a link to the instance here?

Also, to be clear maybe we say something like:
```suggestion
 Currently data from our BigBlueButton is not backed up to another server.
```

Reply:
How about this?

```suggestion
Our [BigBlueButton server](https://bbb.hypha.coop) keeps a database for registered users and room information. This information is not backed up.

Uploaded presentations are kept for 5 days after you have ended your meeting and log history is kept for 28 days. These are BigBlueButton's default configurations and automated purging is performed by the software.
```

Just to confirm, the purge tasks are part of BBB config and not done via a custom script or manually right? Mentioning this may be informative.

Reply:
Please also review https://bigbluebutton.org/privacy/ to see if we have missed anything.

Reply:
The purge tasks is part of BBB's builtin script. There are still standard system log but not sure if we should mention that to?